test/cppsuite/test_harness/workload/database_operation.cxx

```diff
@@ -263,21 +272,21 @@ database_operation::update_operation(thread_context *tc)
         /* Choose a random key to update. */
         uint64_t key_id =
           random_generator::instance().generate_integer<uint64_t>(0, coll.get_key_count() - 1);
-        bool successful_update = tc->update(cursor, coll.id, tc->key_to_string(key_id));
+        bool rollback_required = tc->update(cursor, coll.id, tc->key_to_string(key_id));
 
         /* Reset our cursor to avoid pinning content. */
         testutil_check(cursor->reset(cursor.get()));
```

Comment:
Do we need to reset cursor in case of rollback? As far as I know, rollback will reset all the cursors for you.

Reply:
I think it will handle it for you but we want to reset anyway.